src/file-system-cache-blob-storage.js

```diff
@@ -0,0 +1,93 @@
+var fs = require('fs-plus')
+var path = require('path')
+
+module.exports = (function () {
+  FileSystemCacheBlobStorage.load = function (directory) {
```

Comment:
Good call. :+1: I think I still like the explicitness of mentioning that this is a file-system cache, though: so, what about naming it `FileSystemBlobStore`?